src/ert/config/parameter_config.py

```diff
@@ -46,11 +47,17 @@ def parse_config(
     return args, kwargs
 
 
+class ParameterSource(StrEnum):
+    forward_init = auto()
```

Comment:
This should be changed to: `user_provided`, and we need to keep `forward_init` as is